DVR/URLRequest.swift

```diff
@@ -25,6 +25,15 @@ extension NSURLRequest {
 }
 
 
+extension NSURLRequest {
+    func requestByAppendingHeaders(headers: [NSObject: AnyObject]) -> NSURLRequest {
+        let request = mutableCopy() as! NSMutableURLRequest
```

Comment:
Is this because Foundation just returns an AnyObject here and not the correct mutable type?

If so, is it worth force-unwrapping this, rather than `guard` & `fatalError` or something more descriptive?

Not a big deal, just trying to minimize force-unwraps in general, I'm sure this will succeed 100% of the time.